### PR TITLE
update lr scheduler

### PR DESCRIPTION
**Pull request by Songyuanwei**
add lr scheduler: "constant", "cosine_decay", "polynomial_decay", "multi_step_decay".

### CODE REVIEW

File: examples/stable_diffusion_v2/ldm/modules/train/learningrate.py
Comment (SamitHuang) [inline examples/stable_diffusion_v2/ldm/modules/train/learningrate.py:1]: rename this file to lr_schedulers.py
remove previous `LearningRate` class.

**Review by SamitHuang — APPROVED** [review]

**Review by VictorHe-1 — APPROVED** [review]
